src/pymovements/gaze/gaze_dataframe.py

```diff
@@ -792,6 +792,70 @@ def columns(self) -> list[str]:
         """List of column names."""
         return self.frame.columns
 
+    def map_to_aois(
+        self,
+        aoi_dataframe: pm.stimulus.TextStimulus,
+        *,
+        eye: str = 'auto',
+    ) -> None:
+        """Map gaze data to aois.
+
```

Comment:
maybe add the comment that we treat the aois as (<= >=); so give a comment on the boundaries